src/apps/loan-list/list/loan-list-items.tsx

```diff
@@ -41,10 +32,11 @@ const LoanListItems: FC<LoanListItemProps> = ({
       >
         {t("loanListDueDateModalAriaLabelText")}
       </div>
-      {view === "stacked" &&
-        localDueDates.map((uniqueDueDate: string | null) => {
+      {view === "stack" &&
+        dueDates &&
+        dueDates.map((uniqueDueDate: string) => {
```

Comment:
not fund of those untyped date formats, but we have talked about that ;) 
It's a future todo